### PR TITLE
add getUserProfilePhotos method

### PR DESCRIPTION
**Pull request by feralheart**
I made these changes for the Issue #33 
Documentation:

- [getUserProfilePhotos](https://core.telegram.org/bots/api#getuserprofilephotos)
- [getFile](https://core.telegram.org/bots/api#getfile)

### CODE REVIEW

**Comment by mpociot** [comment]
@feralheart Thank you for the PR. Hm I wonder if we should add this to the Telegram User object.

**Comment by feralheart** [comment]
@mpociot You're welcome. I thinked about it, too, but I didn't want to break the `getUser` method.

**Comment by mpociot** [comment]
I don't think that we break it. The Telegram driver already has a custom user object that gets returned: https://github.com/botman/driver-telegram/blob/master/src/Extensions/User.php
So we could add a setter / getter for the image and pass the image to the object when it gets created.

**Comment by feralheart** [comment]
It's a good idea. I will do it :)

File: src/TelegramDriver.php [hunk 1]
```diff
@@ -71,6 +71,43 @@ public function getUser(IncomingMessage $matchingMessage)
...
+                $profilePhotoParameters = [
+                    'file_id' => $photo['file_id'],
+                ];
+                $responsePhotoFile = $this->http->post($this->buildApiUrl('getFile'), [], $profilePhotoParameters);
+
+                $responseDataPhotoFile = json_decode($responsePhotoFile->getContent(), true);
+                if ($responsePhotoFile->getStatusCode() !== 200) {
+                    throw new TelegramException('Error retrieving user photos info: '.$responseDataPhotoFile['description']);
+                }
+                $userProfilePhotoPaths[] = 'https://api.telegram.org/file/bot'.env('TELEGRAM_TOKEN').'/'.$responseDataPhotoFile['result']['file_path'];
```
Comment (kubk) [inline src/TelegramDriver.php]: Environment variable 'TELEGRAM_TOKEN' is hardcoded here. Maybe it is better to retrieve the token from $this->config?

**Comment by antimech** [comment]
@mpociot so? Merge maybe?

**Comment by antimech** [comment]
@feralheart any upd?

**Comment by filippotoso** [comment]
Move this to the User object, replace the env() call with $this->config() and I'll approve it.

**Pull request closed without merging**